lib/bigquery/index.js

```diff
@@ -96,15 +96,10 @@ var SCOPES = ['https://www.googleapis.com/auth/bigquery'];
  */
 function BigQuery(options) {
   if (!(this instanceof BigQuery)) {
+    options = util.normalizeArguments(this, options);
```

Comment:
That won't work either, the trade off for killing the factory patterns is that we can't store the global config in a closure any longer.

https://github.com/callmehiphop/gcloud-node/blob/config-refactor/lib/index.js#L255

So if you move that line down.. you lose the global config.